client/tests/integration/components/project-geography-test.js

```diff
@@ -0,0 +1,210 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import {
+  render,
+  click,
+  fillIn,
+  triggerKeyEvent,
+} from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+
+module('Integration | Component | project-geography', function(hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+
+  test('user can search and add a new bbl', async function(assert) {
+    // array of bbl objects is dcp_dcp_projectbbl_dcp_pasform
+    this.server.create('bbl', {
+      dcpBblnumber: '3071590111',
+      dcpDevelopmentsite: null,
+      dcpPartiallot: null,
+    });
+
+    this.server.create('bbl', {
+      dcpBblnumber: '3071590115',
+      dcpDevelopmentsite: null,
+      dcpPartiallot: null,
+    });
+
+    const bbl1Model = await this.owner.lookup('service:store').findRecord('bbl', 1);
+    const bbl2Model = await this.owner.lookup('service:store').findRecord('bbl', 2);
+
+    const arrayOfBbls = [bbl1Model, bbl2Model];
+
+    this.bbls = arrayOfBbls;
+
+    await render(hbs`
+      <ProjectGeography
+        @bbls={{this.bbls}} />
+    `);
+
+    // labs-ember-search class for search input
+    await fillIn('.map-search-input', '1000120001');
+    await triggerKeyEvent('.labs-geosearch', 'keypress', 13);
+
+    assert.dom('[data-test-bbl-title="1000120001"]').exists();
+
+    await fillIn('.map-search-input', '1000030001');
+    await triggerKeyEvent('.labs-geosearch', 'keypress', 13);
+
+    assert.dom('[data-test-bbl-title="1000030001"]').exists();
```

Comment:
Beautiful.

Reply:
Question: does this assert that the BBL exists in the DOM?

Reply:
It checks that the list of bbls displays one bbl that has a bbl.dcpBblnumber = 1000030001